cmd/kubelet/app/server.go

```diff
@@ -1193,9 +1193,7 @@ func startKubelet(k kubelet.Bootstrap, podCfg *config.PodConfig, kubeCfg *kubele
 	if kubeCfg.ReadOnlyPort > 0 {
 		go k.ListenAndServeReadOnly(netutils.ParseIPSloppy(kubeCfg.Address), uint(kubeCfg.ReadOnlyPort))
 	}
-	if utilfeature.DefaultFeatureGate.Enabled(features.KubeletPodResources) {
-		go k.ListenAndServePodResources()
-	}
+	go k.ListenAndServePodResources()
```

Comment:
Should this be done in 1.29?

Reply:
AFAIK the change with locks the FG to on is also supposed to remove the corresponding if()s - we did so when promoting cpumanager/devicemanager and if memory serves in other GA graduations, but if I got it wrong I'm happy to revert the FG check removals.

Reply:
when locked to default we remove all checks. Lock to default is only for not breaking users passing feature gates into the command line or kubelet config